package.json

```diff
@@ -60,18 +59,17 @@
     "sass": "^1.63.6",
     "sass-loader": "^10.4.1",
     "sinon": "^14.0.2",
-    "storybook": "^7.0.27",
+    "storybook": "^7.5.3",
     "typescript": "4.5.5",
     "vite": "^4.4.4",
     "vite-plugin-environment": "^1.1.3",
+    "vite-plugin-vuetify": "^1.0.2",
     "vitest": "^0.33.0",
-    "vue-class-component": "^7.2.6",
-    "vue-property-decorator": "^9.1.2",
+    "vue-facing-decorator": "^3.0.3",
     "vue-style-loader": "^4.1.3",
     "vue-template-compiler": "^2.7.14",
     "vue-test-utils-helpers": "git+https://github.com/bcgov/vue-test-utils-helpers.git",
-    "vuelidate-property-decorators": "1.0.28",
     "vuex-class": "^0.3.2",
-    "vuex-module-decorators": "^1.2.0"
+    "vuex-module-decorators": "^2.0.0"
```

Comment:
If you manage to remove the vuex import then these last 2 dependencies won't be needed either.

Reply:
Yay!